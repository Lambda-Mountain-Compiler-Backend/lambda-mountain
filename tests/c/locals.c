
int main() {
   int local_x = 3, local_y, local_z = 4;
}
