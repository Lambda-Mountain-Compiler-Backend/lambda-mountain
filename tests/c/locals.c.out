
int main() {
   int local_x = 3
   int local_y;
   int local_z = 4;
}
